layouts/shortcodes/cve-feed.html

```diff
@@ -21,3 +22,43 @@
   {{ end }}
   </tbody>
 </table>
+
+<script>
+  window.addEventListener("DOMContentLoaded", () => {
+    const table = document.querySelector("table.security-cves")
+    const cveFeedBucket = table.getAttribute("data-cve-feed-bucket")
+    const tbody = table.querySelector("tbody")
+    const cachedInnerHTML = tbody.innerHTML
+  
+    const renderCVEs = (cves) => {
+      let tbodyHTML = ""
+      for (let cve of cves) {
+        tbodyHTML += `<tr>
```

Comment:
Additionally, we should also refresh the table _caption_ when fetching new data, as it provides the timestamp indicating the last update of the CVE feed data.